packages/faustwp-cli/utils/handleTelemetrySubcommand.ts

```diff
@@ -0,0 +1,74 @@
+import chalk from 'chalk';
+import { getCliArgs } from './getCliArgs.js';
+import { userConfig } from './userConfig.js';
+import { styles } from './styles.js';
+
+const TELEMETRY_ENABLED = 'telemetry.enabled';
+const { log } = console;
```

Comment:
This feels like it may add confusion down the road. Any reason why we would not use just `console.log()`?